.travis.yml

```diff
@@ -1,29 +1,44 @@
-language: python
+language: c
+
+os:
+  - osx
+  - linux
+
+compiler:
+  - clang
+  - gcc
 
 env:
-  - TOX_ENV=lint
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=pypy
-  - TOX_ENV=docs
-  - TOX_ENV=docs-spellcheck
-  - TOX_ENV=docs-linkcheck
+  matrix:
+     - TOX_ENV=lint
+     - TOX_ENV=py26
+     - TOX_ENV=py27
+     - TOX_ENV=pypy
+     - TOX_ENV=docs
+     - TOX_ENV=docs-spellcheck
+     - TOX_ENV=docs-linkcheck
 
 matrix:
   allow_failures:
-    - env: "TOX_ENV=docs-linkcheck"
+    - env: TOX_ENV=docs-linkcheck
+  exclude:
+    - os: osx
```

Comment:
https://github.com/rackerlabs/mimic/pull/103#issuecomment-70138476 indicates to me that multi-OS is in fact enabled on this repo so we should in fact be seeing OS X builds in the matrix, and I don't see any, so … I think you're reading it right.  The output from the tests seemed consistent with the labels in the matrix, and it was run 2 days after that comment indicating that the multi-OS support is in fact enabled.

Just to make sure, I re-ran the build to see what happened.  Maybe we need to get in touch with support again?